### PR TITLE
Silencium

### PR DESCRIPTION
**Pull request by XeonFaux**
Hurray!

### CODE REVIEW

File: code/game/mecha/combat/reticence.dm [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	add_req_access = 0
 	internal_damage_threshold = 25
 	max_equip = 2
-	step_energy_drain = 3
+	step_energy_drain = 0 // MIME MECH RUNS ON IMAGINARY POWER.
```
Comment (TDcoolguy300) [inline code/game/mecha/combat/reticence.dm]: Why?

Comment (XeonFaux) [inline code/game/mecha/combat/reticence.dm]: It only makes sense that mimes run on nothing.

Comment (TDcoolguy300) [inline code/game/mecha/combat/reticence.dm]: Balancing wise it's shit.

**Comment by ghost** [comment]
:+1: Dank shit right here. Code looks good

**Comment by Pyko1** [comment]
`+	desc = "A wall with silencium plating. Honk!"`
HONK??!

**Comment by GunHog** [comment]
Hippies know what fun is. Please also consider porting https://github.com/tgstation/-tg-station/pull/14861 in a separate PR. /tg/ might not give this a shot on their codebase, so I am wondering if you guys would give it a trial run.

**Comment by XeonFaux** [comment]
@GunHog , sure thing.

**Comment by Nokington** [comment]
:+1:

**Comment by XeonFaux** [comment]
Now also ports https://github.com/tgstation/-tg-station/pull/14861

**Comment by TDcoolguy300** [comment]
Would of prefered that to of been in it's own PR Xeon.

**Comment by XeonFaux** [comment]
Whateva, this makes mime mech possible. So I just associated the mech medical gun with mechs

**Comment by XeonFaux** [comment]
Reverted mech gun. I'll fix it and put it in it's own PR when I get home

File: code/game/machinery/doors/airlock_types.dm [hunk 2]
```diff
@@ -179,6 +179,16 @@
 	var/mineral = "bananium"
 	doortype = /obj/structure/door_assembly/door_assembly_clown
 
+/obj/machinery/door/airlock/mime
+	name = "silencium airlock"
+	desc = "The exact opposite of a squeaky door."
+	icon = 'icons/obj/doors/airlocks/station/public.dmi'
+	color = "#87878715"
```
Comment (sp1d3rx) [inline code/game/machinery/doors/airlock_types.dm:186]: use constants

File: code/game/machinery/doors/airlock_types.dm [hunk 3]
```diff
@@ -179,6 +179,16 @@
...
+	opacity = 0
+	var/mineral = "silencium"
```
Comment (sp1d3rx) [inline code/game/machinery/doors/airlock_types.dm:188]: constants please

File: code/game/objects/structures/false_walls.dm [hunk 4]
```diff
@@ -243,6 +245,15 @@
 	walltype = "bananium"
 	canSmoothWith = list(/obj/structure/falsewall/clown, /turf/simulated/wall/mineral/clown)
 
+/obj/structure/falsewall/mime
+	name = "silencium wall"
+	desc = "A wall with silencium plating."
+	color = "#87878715"
+	opacity = 0
+	icon = 'icons/turf/walls/wall.dmi'
+	mineral = "silencium"
```
Comment (sp1d3rx) [inline code/game/objects/structures/false_walls.dm:254]: should be a constant

File: code/modules/mining/equipment_locker.dm [hunk 5]
```diff
@@ -19,7 +19,7 @@
 	var/ore_pickup_rate = 15
 	var/sheet_per_ore = 1
 	var/point_upgrade = 1
-	var/list/ore_values = list(("sand" = 1), ("iron" = 1), ("gold" = 20), ("silver" = 20), ("uranium" = 20), ("bananium" = 30), ("diamond" = 40), ("plasma" = 40))
+	var/list/ore_values = list(("sand" = 1), ("iron" = 1), ("gold" = 20), ("silver" = 20), ("uranium" = 20), ("bananium" = 30), ("silencium" = 30), ("diamond" = 40), ("plasma" = 40))
```
Comment (sp1d3rx) [inline code/modules/mining/equipment_locker.dm:22]: where are the constants?

**Comment by Carbonhell** [comment]
Why are all the icons effect/empty?I assume you need sprites?If so you should ask fury

**Comment by Pyko1** [comment]
I recall we used to have mime airlock and metal before rebase

**Comment by Nebyab** [comment]
sure

**Comment by Spacedong** [comment]
mime airlock was just a renamed white door

File: code/game/machinery/doors/airlock_types.dm (hunk 2 above)
Comment (danhibiki) [inline code/game/machinery/doors/airlock_types.dm:186]: Should the color var even be that? I was under the assumption it was the standard 6 digit RGB hex value

**Comment by nalarmg** [comment]
Maintainers, is this safe to merge?

**Comment by TDcoolguy300** [comment]
Doesnt appear to be safe, the mime door appears to be missing it's construction stages.

**Comment by nalarmg** [comment]
Pending code review comment addressing before merge.

**Comment by Nebyab** [comment]
once this is confirmed safe, i'd be fine with merging it

**Comment by TDcoolguy300** [comment]
Xeon when you can, could you give us images of the items?

**Comment by GunHog** [comment]
Images...of invisible items....

**Comment by Nokington** [comment]
I guess this is dead.

**Comment by nalarmg** [comment]
No visible attempts at addressing the review items given for over a week. Closing. Feel free to open this again or create a new PR in the future if there's further interest in this.

**Pull request closed without merging**